packages/astro/src/core/errors/overlay.ts

```diff
@@ -522,6 +522,15 @@ class ErrorOverlay extends HTMLElement {
 
 		const el = this.root.querySelector(selector);
 
+		if (html) {
+			// Automatically detect links
+			text = text.split(' ').map(v => {
+				if (!v.startsWith('https://')) return v;
+				if (v.endsWith('.')) return `<a href="${v.slice(0, -1)}">${v.slice(0, -1)}</a>.`
+				return `<a href="${v}">${v}</a>`
```

Comment:
```suggestion
				if (v.endsWith('.')) return `<a target="_blank" href="${v.slice(0, -1)}">${v.slice(0, -1)}</a>.`
				return `<a target="_blank" href="${v}">${v}</a>`
```

This isn't super relevant at the moment, but links shouldn't open in the same tab, or users will lose their HMR state (the see docs link is also `target="_blank"`, so are Markdown links)